manageiq-providers-amazon.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "aws-sdk-servicecatalog",       "~> 1.0"
   spec.add_dependency "aws-sdk-sns",                  "~> 1.0"
   spec.add_dependency "aws-sdk-sqs",                  "~> 1.0"
-  spec.add_dependency "net-scp",                      "~> 1.2"
+  spec.add_dependency "net-scp",                      "~> 4.0"
```

Comment:
Should we loosen this to ">= 1.2", "<5"?

Reply:
Any time I see a change like this with no code changes, I always question whether we're locked down too tightly.

Reply:
It also eases merging

Reply:
Okay, I had wanted to ensure that we pull in a version that has the openssl 3.0 fix but fine if we let core do that I suppose.

Done

Reply:
I was going to let core do that unless it's important here.  It wasn't clear to me that it is important here (since there is not code change).